sdks/python/apache_beam/testing/util.py

```diff
@@ -87,27 +88,56 @@ def __repr__(self):
 
 
 def equal_to_per_window(expected_window_to_elements):
-  """Matcher used by assert_that to check on values for specific windows.
+  """Matcher used by assert_that to check to assert expected windows..
+
+  The 'assert_that' statement must have reify_windows=True. This assertion works
```

Comment:
Given that the matcher has to know if it expects windows to be reified, can we make that happen automatically and remove this setting?

Reply:
Done.